community/news/index.html

```diff
@@ -2,12 +2,42 @@
 layout: community
 title: News
 main-blurb: New releases and other announcements
----          
-                
+---                      
         <!-- begin News -->
         <hr class="whitespace">
         <div id="community-posts" class="row">
-{% for post in site.posts %}
+{% if paginator.total_pages > 1 %}
+        <div class="small-12 medium-8 medium-offset-2 columns">
+        <ul class="pagination">
+          <li>
+          {% if paginator.previous_page %}
+            <a href="{{ paginator.previous_page_path | prepend: site.baseurl | replace: '//', '/' }}">&laquo; Prev</a>
+          {% else %}
+            <span>&laquo; Prev</span>
```

Comment:
not sure these "else" cases are obviously enough inactive because those that are links just look a bit bolder